### DISCUSSION TITLE
개발 참고 유튜브

### DISCUSSION THREAD

**Discussion by rapa1130** [SeoulTechTCPGame/HumanSurvival#68] [General]
뱀서라이크 한국어 강의
https://www.youtube.com/watch?v=MmW166cHj54&list=PLO-mt5Iu5TeZF8xMHqtT_DhAPKmjF6i3x

뱀파이어 서바이벌 외국어 강의
https://www.youtube.com/watch?v=ixSN42SM3gQ&list=PL0GUZtUkX6t7zQEcvKtdc0NvjVuVcMe6U

**Comment by seong0929** [comment]
unity 기초 공부용 영상
https://www.youtube.com/@wergia/featured
